housekeeper/store/api/handlers/update.py

```diff
@@ -59,3 +54,13 @@ def update_finished_retrieval_task(self, retrieval_task_id: int) -> None:
         ).all()
         for archive in completed_archives:
             self.update_retrieval_time_stamp(archive=archive)
+
+    @staticmethod
+    def update_retrieval_task_id(archive: Archive, retrieval_task_id: int):
+        """Sets the retrieval_task_id in the Archive entry for the provided file."""
+        archive.retrieval_task_id = retrieval_task_id
```

Comment:
Do we care about the previous ID in some way for tracking purposes? Is there merit to have a log file for these etc. ? Not sure how these things are handled. Just wondering.

Reply:
If it is data that needs to be retained I think we should store it in the db

Reply:
Well, it is sort of under discussion what to do with samples that are retrieved multiple times. The latest suggestion is that we only keep track of the latest retrieval/archivation but another possibility is to allow multiple entries in the Archive table for a single file, each keeping track of one job.

Reply:
If we decide to change, it will be in a different PR.